galaxy_skeleton2.R

```diff
@@ -92,15 +93,30 @@ comma_separated_values_with_names <-
     comma_separated_values)    
 
 # Replace the question mark and come up with a file name
-cat(?, sep = "\n", file = "?")
+cat(comma_separated_values_with_names, sep = "\n", file = "processed_data.txt")
 
 # Read the file back in as a normal csv-file. The readr-package is part of
 # tidyverse, so it is already loaded.
-galaxies <- read_csv("?")
+galaxies <- read_csv("processed_data.txt")
 
 
 # You should now have a nice, clean data frame with galaxies and their
 # characteristics in memory. As of March 2022 it should contain 796
 # observations.
 
+# ------------------------------------------------
+# Problem 3
+
+galaxies %>% filter(D <= 11) %>% ggplot(aes(x = a_26)) + 
```

Comment:
If you made a scatterplot with the diameter against the distance to each galaxy you would see that this statement seems to be correct. This is because it is hard to observe smaller objects that are far away than the ones that are close.